syntax = "proto3";

package fixbug;

option cc_generic_services = true;

message ResultCode
{
  int32 errcode = 1;
  bytes errmsg = 2;
}

message LoginRequest
{
  bytes name =1;
  bytes pwd = 2;
}

message LoginRsponse
{
  ResultCode result = 1;
  bool success = 2;
}

message RegisterRequest
{
  uint32 id =1;
  bytes name =2;
  bytes pwd = 3;
}

message RegisterResponse
{
  ResultCode result = 1;
  bool success = 2;
}

service USerServiceRpc
{
  rpc Login(LoginRequest) returns(LoginRsponse);
  rpc Register(RegisterRequest) returns(RegisterResponse);
}


